recipes/bash_kernel/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "bash_kernel" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a08c84eddd8179de5234105821fd5cc210015671a0bd3cd0bc4f631c475e1670
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m flit install --deps none
+    - {{ PYTHON }} -m bash_kernel.install --prefix {{ PREFIX }}
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+    - flit
+    - ipykernel
+  run:
+    - python >=3.0
+    - ipykernel
+    - pexpect
```

Comment:
This seems like it should depend on bash

Reply:
But bash is generally available under mac and linux, why do we need `bash` from conda?

Reply:
I did not knew Bash was available from conda-forge. I guess if we follow the spirit of conda, this package should avoid dependency on system packages, including this one. Makes sense.

Reply:
Just added `bash` as `run` requirement.

Reply:
https://github.com/conda-forge/bash_kernel-feedstock/commit/caa8fbd62d626ebc74ea31479fe9569749bead48

Reply:
How about removing `noarch` and using https://github.com/conda-forge/git-bash-feedstock/blob/master/recipe/meta.yaml on Windows?

Reply:
I tried but bash_kernel could not start with git-bash. I did not try to debug though.

Reply:
Okay, then I think we need to remove bash here or remove noarch and skip windows.  https://github.com/conda-forge/bash-feedstock/blob/6e15f814dd32ec95a449a48a63f2762dd019e06a/recipe/meta.yaml#L22

Reply:
@blink1073 it was discussed above with @hmaarrfk

Reply:
I think now that we are explicitly requiring bash conda would fail to solve on Windows even if you have bash installed.  I'd say remove bash here.

Reply:
Are you referring to bash from linux subsystem under windows 10? Since the recipe fails with or without `bash` requirement, I tend to keep it as is until we find a way to make `bash_kernel` works under windows.

Reply:
Yeah, WSL isn't there in the CI.  Seems odd to me to preclude it from being used there for an ideal of having allthethings on conda-forge.

Reply:
actually, *from* wsl you'd be identified as linux anyway.  sorry for the noise.

Reply:
```
(base) C:\Users\mark>conda create --name bash
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\mark\Miniconda3\envs\bash



Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate bash
#
# To deactivate an active environment, use
#
#     $ conda deactivate


(base) C:\Users\mark>bash
'bash' is not recognized as an internal or external command,
operable program or batch file.

(base) C:\Users\mark>conda activate bas
Could not find conda environment: bas
You can list all discoverable environments with `conda info --envs`.


(base) C:\Users\mark>conda activate bash

(bash) C:\Users\mark>conda install m2-bash
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\mark\Miniconda3\envs\bash

  added / updated specs:
    - m2-bash


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    m2-bash-4.3.042            |                5         3.5 MB
    ------------------------------------------------------------
                                           Total:         3.5 MB

The following NEW packages will be INSTALLED:

  m2-bash            pkgs/msys2/win-64::m2-bash-4.3.042-5
  m2-msys2-runtime   pkgs/msys2/win-64::m2-msys2-runtime-2.5.0.17080.65c939c-3
  msys2-conda-epoch  pkgs/msys2/win-64::msys2-conda-epoch-20160418-1


Proceed ([y]/n)? y


Downloading and Extracting Packages
m2-bash-4.3.042      | 3.5 MB    | ############################################################################ | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(bash) C:\Users\mark>bash
bash.exe: warning: could not find /tmp, please create!
bash-4.3$
```
you might as well depend on `m2-bash` on windows.

Reply:
OK, I spent more time on `bash_kernel` under windows. There appears to be two problems, one with conda, and one with `bash_kernel` itself that prevents the use of `bash_kerne` under windows. I have submitted to bug reports 

https://github.com/conda-forge/staged-recipes/issues/9465
https://github.com/takluyver/bash_kernel/issues/96